### ISSUE TITLE
A11y ARIA Role Warning 

### ISSUE THREAD

**Issue by n00ki** [carstenlebek/svelte-email#18]
Hi :)

Lately, I noticed an a11y warning in my tests regarding `svelte-email`: `A11y: <td> cannot have role 'presentation'`.
This warning comes from the `<td>` element in `Column.svelte`. 
Maybe we should consider changing the aria role to `cell`?
```svelte
<td style={styleToString(styleDefault)} role="cell" {...$$restProps}>
```

Thanks in advance🙏

**Comment by kengher** [comment]
Bump.
role='presentation' is breaking run build.

**Comment by polaroi8d** [comment]
> Bump. role='presentation' is breaking run build.

It just drops a warning, no?
